internal/pkg/bulk/bulk_remote_output_test.go

```diff
@@ -148,3 +153,57 @@ func Test_CreateAndGetBulkerChanged(t *testing.T) {
 	assert.Nil(t, err)
 	assert.Equal(t, true, cancelFnCalled)
 }
+
+func Benchmark_CreateAndGetBulker(b *testing.B) {
+	b.Skip("Crashes on remote runner")
```

Comment:
I'm not sure why, but this causes issues when `make benchmark` is ran, but these tests can be ran individually without this